cmd/influx/config/config.go

```diff
@@ -14,15 +14,18 @@ import (
 
 // Config store the crendentials of influxdb host and token.
 type Config struct {
+	Name string `toml:"-" json:"-"`
```

Comment:
why are these not getting encoded?

Reply:
we always export them as map[name]->config, this will prevent user edit any bad data.

Reply:
I want to make sure I understand what you are saying. You are saying that b/c the config has a name as the `[name]` object identifier in the `.toml` file, we have to remove it from being added to b/c they could edit one and not the other yah?

Reply:
yeah, that's one reason